### DISCUSSION TITLE
Certificates disappear

### DISCUSSION THREAD

**Discussion by elmasoki** [argoproj/argo-cd#8295] [Q&A] [answered]
After Argocd openshift operator  0.2.0  upgrade , TLS definitons in the argocd-tls-certs-cm configmap disappear automatically.

**Comment by chetan-rns** [comment, marked as answer]
Hi @elmasoki, there is an open ticket for this issue along with the workaround https://github.com/redhat-developer/gitops-operator/issues/271

**Reply by elmasoki** [reply]
thank you
